src/sass/components/_ballotList.scss

```diff
@@ -17,7 +17,23 @@
   background: adjust_hue(lighten(#f5f5f5, 1), 0);
 }
 
+// Ballot (Voter Guide) Item
+
+@include media-object('.ballot-item', '.ballot-item__avatar', '.ballot-item__content');
+
+
 .ballot-item {
-  min-height: 85px;
-  position: relative;
+  @include list-item-card;
```

Comment:
Eventually, we can find better ways to consolidate similar styles, but using mixins to keep track of similarly styled elements seemed reasonable for the time being.